packages/frontpage/app/(app)/actions.tsx

```diff
@@ -17,7 +17,7 @@ export async function getMorePostsAction(cursor: number) {
             id={post.id}
             title={post.title}
             url={post.url}
-            votes={post.voteCount}
+            votes={post.voteCount == 0 ? 1 : post.voteCount}
```

Comment:
I agree with not returning `voteCount: 1` when nobody has created any vote records ie. I agree with returning a 0 count by default. But we shouldn't have to be defaulting to 1 in 4 different places in the view layer either. Not sure what the solution is yet but I'll have a think about it.

The only solution I have so far is adding a layer between the data layer and components, but I'm not sure where this should go. For pragmatism it might just make sense to just do `voteCount: 1` from the data layer for now and add the extra layer later when we create the API.

Reply:
This will be fixed when we create a vote on users' behalves 😘

Reply:
This is actually fixed now without the future update.